test/end-to-end/getRedirectionURL.test.js

```diff
@@ -200,7 +200,6 @@ describe("getRedirectionURL Tests", function () {
                 if (!_isPasswordlessSupported) {
                     didSkip = true;
                     this.skip();
-                    return;
```

Comment:
why is this removed?

Reply:
it was removed by the linter. Having a return here makes no difference as the test is skipped completely after calling `this.skip()`. Nevertheless, I have put it back to avoid confusion.